### DISCUSSION TITLE
Partials/includes of other markdown content in a index file?

### DISCUSSION THREAD

**Discussion by kaosine** [vuepress/core#1097] [Q&A]
This is probably a weird question, but is there any way to import content from another file into an index file of a section?

Basically trying to do this:
```
|-docs
|--user-guide
|---sections
|----getting-started
|-----sections
|------sectiona.md
|------sectionb.md
|----index.md <- this would be the main file w/a intro and fragment calls
|                         in it to the sections part
|---index.md <-user-guide index file
|-index.md <- documentation root index file
```
We're just trying to modularize our content, so it's easier to modify things w/out making the section like getting-started too unwieldy. That way, if section b, or even d needs updating, it's easy to go to that file instead of putting it all into a massive single file and trying to get down to the part you need to edit.

I tried adding the vuepress-plugin-md-enhance with the current latest and all I get on the page is `@md` on the page. With the `md` being a link to the page instead of what is expected.

**Comment by Mister-Hope** [comment]
You may need to explain why mdImport not work in your situation.

**Reply by kaosine** [reply]
Because all it did was insert a link with a @ in front of it, with the text md as the link. I added your plugin, but it didn't do anything other than that. Wasn't sure if I needed to add something to our config file or what.

**Reply by Mister-Hope** [reply]
https://vuepress-theme-hope.github.io/v2/md-enhance/guide/include.html Have you read the docs?

**Reply by kaosine** [reply]
I honestly couldn't find them, but I'll read through that now. Thanks for that link

**Reply by Mister-Hope** [reply]
Just a kind note, if you want docs, search them on npm first (https://www.npmjs.com/package/vuepress-plugin-md-enhance)

Then try homepage or readme:

<img width="977" alt="image" src="https://user-images.githubusercontent.com/33315834/192004494-bda24257-07c1-4090-a6c7-f94f5be40cb6.png">
